src/XunitToMSTestConverter.CodeFixes/Helpers/SyntaxNodeExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Reflection;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Simplification;
+
+namespace XunitToMSTestConverter.Helpers;
+
+internal static class SyntaxNodeExtensions
+{
+    private static Optional<SyntaxAnnotation?> s_addImportsAnnotation;
+
+    private static SyntaxAnnotation? AddImportsAnnotation
+    {
+        get
+        {
+            if (!s_addImportsAnnotation.HasValue)
+            {
+                var property = typeof(Simplifier).GetTypeInfo().GetDeclaredProperty("AddImportsAnnotation");
```

Comment:
This became public in Roslyn. roslyn-analyzers repo still relies on reflection as it uses a super old version of Roslyn IIRC